static/css/business.less

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+   cfgov-refresh
+   Business with us pages at /doing-business-with-us/.
+   ========================================================================== */
+
+// Used to set padding to 30px
+@padding: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
What about something like:
```
@grid_gutter-width: 30px
@grid_gutter-width__em: unit(@grid_gutter-width / @base-font-size-px, em);
```
(please critique the BEM style)

Is the concern though that `@grid_gutter-width` may load before `@base-font-size-px` since they are in different CF repos? I wasn't necessarily thinking a new variable for CF, but perhaps a top-level variable in FJ for handling the implementation of the CF `@grid_gutter-width` in a consistent way across different Less files.

(As an aside, is there consistency with the CF variables containing or not containing units in the name?)